app/src/main/java/com/keylesspalace/tusky/ComposeActivity.java

```diff
@@ -1158,6 +1158,33 @@ private void addMediaToQueue(@Nullable String id, QueuedMedia.Type type, Bitmap
         }
     }
 
+    private void updateContentDescriptionForAllImages() {
+        List<QueuedMedia> items = new ArrayList<>(mediaQueued);
+        for (QueuedMedia media : items) {
+            updateContentDescription(media);
+        }
+    }
+
+    private void updateContentDescription(QueuedMedia item) {
+        if (item.preview != null) {
+            String imageId;
+            if (!TextUtils.isEmpty(item.description)) {
+                imageId = item.description;
+            } else {
+                int idx = getImageIdx(item);
+                if (idx < 0)
+                    imageId = getString(R.string.compose_preview_image_description_image_unknown);
```

Comment:
Is this a case that can happen? When?

Reply:
If item has the `description` - app set this description as `content description`
If item has not an item description - app set its `index+1` as description, but if item not found at the queue - it set to `unknown`. Generally, it should never happening, but, i prefer to handle all possible cases even they are not possible now, but it may possible in the future after some behaviour change

Reply:
I would just set un-set the description in that case, no need for an additional string resource. Keep in mind it needs to be translated into 28 languages.

Reply:
Ah, got it. Thanks, i will update code now